codex-rs/tui/src/app.rs

```diff
@@ -274,6 +276,25 @@ impl App<'_> {
                 }
                 AppEvent::KeyEvent(key_event) => {
                     match key_event {
+                        KeyEvent {
+                            code: KeyCode::Char('e'),
+                            modifiers: crossterm::event::KeyModifiers::CONTROL,
+                            kind: KeyEventKind::Press,
+                            ..
+                        } => {
+                            let env = std::env::var("MANUALLY_DEBUG_TUI_BACKGROUND_RETRY")
+                                .unwrap_or_default();
+                            if env == "1" {
```

Comment:
```suggestion
                        } if std::env::var("MANUALLY_DEBUG_TUI_BACKGROUND_RETRY").is_ok_and(|v| v == "1") => {
                            // ...
```

then you don't have to repeat the dispatch_key_event logic.